designer/server/src/main/scala/pl/touk/nussknacker/ui/server/AkkaHttpBasedRouteProvider.scala

```diff
@@ -391,11 +399,12 @@ class AkkaHttpBasedRouteProvider(
         new NuDesignerExposedApiHttpService(
           appApiHttpService,
           componentsApiHttpService,
-          userApiHttpService,
+          managementApiHttpService,
+          nodesApiHttpService,
           notificationApiHttpService,
           scenarioActivityApiHttpService,
           scenarioParametersHttpService,
-          nodesApiHttpService
+          userApiHttpService
```

Comment:
Why the order is changed?

Reply:
I have alligned components alphabetically to easily check order.